bin/init.sh

```diff
@@ -17,7 +17,10 @@ ln -sf "$(pwd)/bazel-genfiles/external/io_istio_api/proxy/v1/config/cfg.pb.go" \
   vendor/istio.io/api/proxy/v1/config/cfg.pb.go
 mkdir -p vendor/istio.io/manager/test/grpcecho
 ln -sf "$(pwd)/bazel-genfiles/test/grpcecho/echo.pb.go" \
-  vendor/istio.io/manager/test/grpcecho/
+   vendor/istio.io/manager/test/grpcecho/
+mkdir -p vendor/github.com/googleapis/googleapis/google/rpc
+ln -sf $(pwd)/bazel-genfiles/external/com_github_googleapis_googleapis/google/rpc/*.pb.go \
```

Comment:
can you also make sure `bin/gazelle` works with this change and do a fix if it doesn't?